### PR TITLE
List all doctors features

### PR DESCRIPTION
**Pull request by micheaol**
## Changes introduced via this PR: 👇 

- [x]  When the user clicks the "My appointments" link in the navigation panel they can see a list of their appointments (with information about doctor's name, date, and city).
- [x]  Show and style all the doctors' cards

### CODE REVIEW

File: src/pages/Appointments/Appointments.jsx [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+
+const Appointment = () => {
+  const appointment = useSelector((state) => state.appointments);
+  const doctors = useSelector((state) => state.doctor);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchAp(2));
+    dispatch(fetchD());
+  }, [dispatch]);
```
Comment (AlexRS90) [inline src/pages/Appointments/Appointments.jsx:16]: When I visited the appointments page, it's not showing any card with the information about the appointments, the console is showing the next error:
Captura de Pantalla 2022-04-04 a la(s) 17 55 30

And the API has a record that should be displayed!
Captura de Pantalla 2022-04-04 a la(s) 17 59 34



Kindly fix it!

File: src/pages/Doctors/DoctorCard.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+              <h1 className="font-bold text-xl mb-2">{doc.name}</h1>
+              <h1 className="overline p-3 font-bold text-xl mb-2">Experience: {doc.experience} years</h1>
+              <div className="text-gray-700 text-base">
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
+              </div>
+            </div>
+            <div className="px-6 pt-4 pb-2">
+              <span className="cursor-pointer inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2"><FaFacebookF size={20} /></span>
+              <span className="cursor-pointer inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2"><FaTwitter size={20} /></span>
+              <span className="cursor-pointer inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2"><FaLinkedinIn size={20} /></span>
```
Comment (AlexRS90) [inline src/pages/Doctors/DoctorCard.jsx]: I think is a good idea to make work these buttons.
Kindly add the links to the social networks' pages 🤓

**Review by AlexRS90 — CHANGES_REQUESTED** [review]
## Hello @micheaol 
## Congratulations you have done an amazing job so far, but still, there are minor issues to fix!

Highlights:

✅  Profesional Repo
✅  Descriptive commit messages
✅  No linters errors
✅  Doctors' data fetched correctly from the API

Keep rocking 🎸  🎸

**Comment by Amrhub** [comment]
Thank you @AlexRS90  for the review also I checked the UI design, isn't supposed to look like the Behance design,
I mean with the carousel and all of that 🤔?
image

If we have to make it a carousel, here is also a resource we could use to see how an element would look like using tailwind [tailwind carousel elements](https://tailwind-elements.com/docs/standard/components/carousel/)

File: src/pages/Appointments/Appointments.jsx (hunk 1 above)
Comment (micheaol) [reply src/pages/Appointments/Appointments.jsx:16]: @AlexRS90 ,
Thank you for your review. Please note that the path to get an appointment is `http://localhost:3000/api/v1/:user_id/appointments` and at the moment there is no way to get the current user until the `user ui` is done with. Therefore, in the `src/pages/Appointments/Appointments.jsx` we need to `hard-code` `user_id`. This is the reason you are getting the error. Kindly pass in the `user_id` into the `fetchApp` method and the error will go away. Thank you.

File: src/pages/Doctors/DoctorCard.jsx (hunk 2 above)
Comment (micheaol) [reply src/pages/Doctors/DoctorCard.jsx]: @AlexRS90 ,
This will be worked on. Thanks

File: src/pages/Appointments/Appointments.jsx (hunk 1 above)
Comment (AlexRS90) [reply src/pages/Appointments/Appointments.jsx:16]: Hello, @micheaol yeah, you're right, it was my bad, I hard-coded the user-id and the app is fetching the data as expected, but now is throwing a warning! ⚠️ see the picture below ⬇️ 

Captura de Pantalla 2022-04-04 a la(s) 22 20 12

Comment (micheaol) [reply src/pages/Appointments/Appointments.jsx:16]: @AlexRS90 ,

Thank you for the review.
The key issue will be fixed.

File: src/pages/Doctors/DoctorCard.jsx (hunk 2 above)
Comment (AlexRS90) [reply src/pages/Doctors/DoctorCard.jsx]: Hi @micheaol, the buttons are still not working, kindly add the links

**Comment by AlexRS90** [comment]
Hello @micheaol nice implementation adding the buttons to see all doctors' cards, but there is a bug because I'm seeing the cards in this way ⬇️  ⬇️
Captura de Pantalla 2022-04-05 a la(s) 8 36 40

**Review by AlexRS90 — APPROVED** [review]
Hello, @micheaol 

Your project is complete, feel free to merge it!

**Comment by Amrhub** [comment]
The styling phase for doctors will be worked on later on!

**Pull request merged**